opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -45,6 +49,8 @@
     get_aggregated_resources,
 )
 
+# pylint: disable=protected-access
```

Comment:
Why do you need this line?

Reply:
Thanks for the feedback. i've removed it.